src/main/java/com/example/demo/database/Customers.java

```diff
@@ -0,0 +1,48 @@
+package com.example.demo.database;
+
+import jakarta.persistence.*;
+
+@Entity
+
+public class Customers {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "customer_id", nullable = false)
+    private long customer_id;
```

Comment:
You should use the Java naming convention for entity fields. 
So it would be `customerId`
Hibernate will translate this field using `NamingStrategy` to the column name.
So in the DB this column's name will be a `customer_id`